### ISSUE TITLE
About the loss of cl_train

### ISSUE THREAD

**Issue by sherlhw** [haon-chen/MoCa#5]
Hi! I would like to ask why the training loss remains unchanged when I am conducting contrastive learning fine-tuning using my own data?
